src/arload.py

```diff
@@ -115,6 +116,7 @@ def optarg(argflag, argv, pypname):
         opt, arg = getopt.getopt(argv[1:], 'hc:v:', ['help',
```

Comment:
Following on from the comment below (on line 128), this line should read:

opt, arg = getopt.getopt(argv[1:], 'hmc:v:', ['help', 'use_masters',

i.e. the use_masters should match the same index where the 'm' appears in the second argument of this function.

Also, you should update the function armsgs.usage() to include a description of this new command line argument.